### PR TITLE
Some fixes for the recent settings menu.

### PR DESCRIPTION
**Pull request by dclstn**
- fix default highlight keywords when user isnt logged
- fix ambigious descriptions
- fix default friends list sidebar flag

### CODE REVIEW

File: src/modules/settings/components/settings/AutoPlay.jsx [hunk 1]
```diff
@@ -19,15 +19,15 @@ function AutoplayModule() {
           onChange={(value) => setAutoplay(value.length > 0 ? value.reduce((a, b) => a | b) : 0)}>
           <Checkbox key="fpPlayer" value={AutoPlayFlags.FP_VIDEO}>
             <p>Front-page Player</p>
-            <p className={styles.description}>Disables autoplaying streams on the homepage</p>
+            <p className={styles.description}>Autoplays streams on the homepage</p>
```
Comment (night) [inline src/modules/settings/components/settings/AutoPlay.jsx:22]: Copy updates:

> Control Auto Play on various Twitch video players.

> Enable auto play for homepage video player

> Enable auto play for channel hosting video player

> Enable auto play of recommended videos on VoDs

File: src/modules/settings/components/settings/ChannelPoints.jsx [hunk 2]
```diff
@@ -19,15 +19,15 @@ function ChannelPointsModule() {
           onChange={(value) => setChannelPoints(value.length > 0 ? value.reduce((a, b) => a | b) : 0)}>
           <Checkbox key="channelPoints" value={ChannelPointsFlags.CHANNEL_POINTS}>
             <p>Channel Points</p>
-            <p className={styles.description}>Hides channel points from the chat UI to reduce clutter</p>
+            <p className={styles.description}>Show channel point related features</p>
```
Comment (night) [inline src/modules/settings/components/settings/ChannelPoints.jsx:22]: > Enable channel points in the chat UI

> Enable channel point highlighted messages

File: src/modules/settings/components/settings/Chat.jsx [hunk 3]
```diff
@@ -19,27 +19,27 @@ function ChatModule() {
           onChange={(value) => setChat(value.length > 0 ? value.reduce((a, b) => a | b) : 0)}>
           <Checkbox key="chatReplies" value={ChatFlags.CHAT_REPLIES}>
             <p>Chat Replies</p>
-            <p className={styles.description}>Hides the click to reply button in chat</p>
+            <p className={styles.description}>Enables the click to reply button</p>
```
Comment (night) [inline src/modules/settings/components/settings/Chat.jsx:22]: > Show the click to reply button in chat

> Show bits in the chat window

> Show clip embeds in the chat window

> Show new viewer greetings in the chat window

> Show subs, re-subs, and gift subs notices in the chat window

> Show alerts above chat window for hype trains, community chest, etc.

File: src/modules/settings/components/settings/Sidebar.jsx [hunk 4]
```diff
@@ -19,19 +19,19 @@ function SidebarComponent() {
           onChange={(value) => setSidebar(value.length > 0 ? value.reduce((a, b) => a | b) : 0)}>
           <Checkbox key="friends" value={SidebarFlags.FRIENDS}>
             <p>Friends List</p>
-            <p className={styles.description}>Hides the friends list in the left sidebar</p>
+            <p className={styles.description}>Show the friends list in the left sidebar</p>
           </Checkbox>
           <Checkbox key="featuredChannels" value={SidebarFlags.FEATURED_CHANNELS}>
             <p>Featured Channels</p>
-            <p className={styles.description}>Removes the recommended channels in the sidebar</p>
+            <p className={styles.description}>Enable the recommended channels in the sidebar</p>
```
Comment (night) [inline src/modules/settings/components/settings/Sidebar.jsx:26]: > Show friends in the sidebar

> Show recommended channels in the sidebar

> Show recommended friends in the sidebar

> Show offline followed channels in the sidebar

File: src/modules/settings/components/settings/Usernames.jsx [hunk 5]
```diff
@@ -19,17 +19,15 @@ function UsernamesModule() {
           onChange={(value) => setUsernames(value.length > 0 ? value.reduce((a, b) => a | b) : 0)}>
           <Checkbox key="localized" value={UsernameFlags.LOCALIZED}>
             <p>Localized Usernames</p>
-            <p className={styles.description}>Hides localized display names in chat</p>
+            <p className={styles.description}>Show localized display names in chat</p>
```
Comment (night) [inline src/modules/settings/components/settings/Usernames.jsx:22]: > Show localized display names in the chat window

> Show username colors in the chat window

> Show username colors with higher contrast (prevents hard to read names)

File: src/constants.js [hunk 6]
```diff
@@ -196,14 +197,17 @@ export const DefaultValues = {
...
-    },
-  },
+  [SettingIds.HIGHLIGHT_KEYWORDS]:
+    twitch.getCurrentUser() != null
+      ? {
+          0: {
+            id: 0,
+            type: Types.MESSAGE,
+            status: null,
+            keyword: twitch.getCurrentUser().name,
```
Comment (night) [inline src/constants.js:207]: if a user is not logged in, isn't `getCurrentUser()` returning `undefined`? That would result in `Uncaught TypeError: Cannot read property 'name' of undefined`

Comment (dclstn) [reply src/constants.js:207]: The ternary before would catch that `getCurrentUser()` != `null`

**Pull request merged**